package.json

```diff
@@ -46,6 +46,8 @@
     "d3-time-format": "^2.0.0",
     "element-resize-detector": "^1.1.6",
     "lodash": "^4.11.0",
+    "moment": "^2.17.1",
```

Comment:
I'm not sure we need moment as a dependency. It looks to me like your usage of it in the PR could fairly easily be replaced by simple utility functions:

```
moment(date)                       -> new Date(date)
moment(date).add(10, 'months')     -> new Date(date).setMonth(date.getMonth() + 10)
moment(date).format('MMM-DD-YYYY') -> would take a couple utility functions, only used in examples
```

etc.

Anyway, let me know if I'm mistaken, just thinking about bundle sizes these days.